go/plugins/ollama/ollama.go

```diff
@@ -344,3 +388,31 @@ func concatMessages(input *ai.GenerateRequest, roles []ai.Role) string {
 	}
```

Comment:
Do we want to subselect parts here also? (parts.IsText test)

Reply:
Good point, I think it doesn't make a difference for images but I updated it anyway for clarity.